bigtable/google/cloud/bigtable/client.py

```diff
@@ -74,6 +82,18 @@ def inner(self):
     return inner
 
 
+def transport(channel=None):
+    def inner(credentials=None, default_class=None):
+        if channel is not None:
+            return default_class(channel=channel)
+        elif credentials is not None:
+            return default_class(credentials=credentials)
+        else:
+            return None
+
+    return inner
+
```

Comment:
Ok, I'll try this and modify and test accordingly.

Reply:
Actually I have to use closer, since class BigtableTableAdminClient in  bigtable_table_admin_client.py uses callable transport with credentials.
So we need to pass function instance which only accepts two parameters, credentails and default class.

if callable(transport):
                self.transport = transport(
                    credentials=credentials,
                    default_class=bigtable_table_admin_grpc_transport.BigtableTableAdminGrpcTransport,
)

Please see [here](https://github.com/googleapis/google-cloud-python/blob/5932539e220a49c462e661115dec775dd10f4f0b/bigtable/google/cloud/bigtable_admin_v2/gapic/bigtable_table_admin_client.py#L192) 
for an example of where an admin transport would be used.

Reply:
Same with BigtableInstanceAdminClient class